modular_sand/code/game/machinery/computer/arcade/tetris.dm

```diff
@@ -13,10 +13,7 @@
 /obj/machinery/computer/arcade/tetris
 	name = "T.E.T.R.I.S."
 	desc = "The pinnacle of human technology."
-	icon = 'icons/obj/computer.dmi'
```

Comment:
Where were our icon for it again?

Reply:
The icon entries were identical to the original `/obj/machinery/computer/arcade` and did not need to be re-defined.